generator/k8s_docs_tools/templates/supported-versions.j2

```diff
@@ -19,12 +19,21 @@ source of the information - please file a bug at
 https://github.com/charmed-kubernetes/kubernetes-docs/issues 
 rather than editing the text -->
 
-Charmed Kubernetes officially supports the three (3) most recent minor releases
-of Kubernetes.
+Charmed Kubernetes officially supports the three most recent minor releases
+of Kubernetes. The support window for Charmed Kubernetes is explained more
+fully on the [Ubuntu release cycle page](/about/release-cycle#canonical-kubernetes-release-cycle).
 
 Current release: **{{ release }}**
 
-Supported releases: **{{ supported_releases_x }}**
+Supported releases (click buttons for more information): 
+
+<div class="row">
+{%- for rel in supported_releases %}
+ <div class="col-2">
+  <span class="u-vertically-center"><a href="/kubernetes/docs/{{rel}}/components" class="p-button--positive">{{rel}}.x</a></span>
+ </div>
+{%- endfor %}
+</div>
```

Comment:
thanks! I would have got around to this...
I think the buttons are far better at drawing attention to the actual versions.